verification/tagTCI.go

```diff
@@ -3,55 +3,218 @@
 package verification
 
 import (
+	"bytes"
+	"crypto/sha256"
+	"crypto/sha512"
 	"errors"
+	"hash"
+
 	//"reflect"
 	"testing"
 )
 
-func TestTagTCI(d TestDPEInstance, client DPEClient, t *testing.T) {
-	useSimulation := false
-	ctx := getInitialContextHandle(d, client, t, useSimulation)
+const defaultCtxTCITag = TCITag(12345)
+const nonExistentTCITag = TCITag(98765)
+const childCtxTCITag = TCITag(34567)
+
+// Check tagTCI command with default context handle.
+func TestTagTCI(d TestDPEInstance, c DPEClient, t *testing.T) {
+	var err error
```

Comment:
1. Removed clubbed test functions to put them as separate test cases, updated comments